consensus/src/main/java/org/ethereum/beacon/consensus/hasher/SSZObjectHasher.java

```diff
@@ -26,22 +26,17 @@ public SSZObjectHasher(SSZHasher sszHasher) {
     this.sszHasher = sszHasher;
   }
 
-  public static SSZObjectHasher create(
+  public static SSZObjectHasher createIncremental(
       SpecConstants constants, Function<BytesValue, Hash32> hashFunction) {
-
-    SSZHasher sszHasher =
-        new SSZBuilder()
-            .withExternalVarResolver(new SpecConstantsResolver(constants))
-            .withIncrementalHasher(true)
-            .buildHasher(hashFunction);
-    return new SSZObjectHasher(sszHasher);
+    return create(constants, hashFunction, true);
   }
 
-  public static SSZObjectHasher create(Function<BytesValue, Hash32> hashFunction) {
-
+  public static SSZObjectHasher create(
```

Comment:
It actually differs about an `incremental` parameter. This parameter accepts either `true` or `false` in case of `create` and always `true` in case of `createIncremental`. Not sure that I understand your concern correctly.